flink-core/src/main/java/org/apache/flink/api/common/operators/base/BulkIterationStrategy.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2015 EURA NOVA.
```

Comment:
I think you need to remove this copyright statement before we can merge the change.